pkg/master/master.go

```diff
@@ -230,7 +230,7 @@ func setDefaults(c *Config) {
 		// We should probably allow this for clouds that don't require NodePort to do load-balancing (GCE)
 		// but then that breaks the strict nestedness of ServiceType.
 		// Review post-v1
-		defaultServiceNodePortRange := util.PortRange{Base: 30000, Size: 2767}
+		defaultServiceNodePortRange := util.PortRange{Base: 30000, Size: 2768}
```

Comment:
at least on my machines /proc/sys/net/ipv4/ip_local_port_range includes 32768, so I think this was right before.